app/javascript/app/components/data-explorer-content/data-explorer-content-selectors.js

```diff
@@ -542,6 +543,15 @@ export const parseData = createSelector([getData], data => {
   return updatedData.map(d => pick(d, whiteList));
 });
 
+export const getFirstColumnHeaders = createSelector([parseData], data => {
```

Comment:
Could this be named something like `getHeadersWithYears` as it describes better what's happening inside the function?